membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -99,6 +99,7 @@ object AccountDetails {
       def externalisePlanName(plan: RatePlan): Option[String] = plan.product match {
         case _: Product.Weekly => if (plan.name.contains("Six for Six")) Some("currently on '6 for 6'") else None
         case _: Product.Paper => Some(plan.name.replace("+", " plus Digital Subscription"))
+        case _: Product.SupporterPlus => if (plan.name.contains("Guardian Weekly")) Some("Supporter Plus with Guardian Weekly") else None
```

Comment:
I don't think MMA needs to know if it is a Domestic or ROW Guardian Weekly?

Reply:
looks like we look for "plus Digi"
https://github.com/guardian/manage-frontend/blob/cb8b4a85ddac0697d461d00ea4463ba98b71686b/client/components/mma/cancel/voucher/VoucherCancellationFlowStart.tsx#L28

and "6 for 6"
https://github.com/guardian/manage-frontend/blob/98923b8d2735de2bd2b4343b5b7701852118c1c6/shared/productResponse.ts#L131

as well as returning it as the last part of something like "Newspaper Delivery - Everyday"
https://github.com/guardian/manage-frontend/blob/c164820979e345e8c079086b884d7fc8e5c9dcd5/shared/productTypes.ts#L471

So as far as I can see it is there's no code that looks at GW at all in this particular field, let alone whether it's domestic or otherwise.

Reply:
it looks like the system relies much more heavily on the "tier" field to work out what funcionality to offer in manage UI .
https://github.com/guardian/manage-frontend/blob/c164820979e345e8c079086b884d7fc8e5c9dcd5/shared/productTypes.ts#L721-L720